src/main/java/leets/crazyform/domain/workspace/usecase/WorkspaceDeletionImpl.java

```diff
@@ -0,0 +1,30 @@
+package leets.crazyform.domain.workspace.usecase;
+
+import leets.crazyform.domain.workspace.exception.WorkspaceDeletionException;
+import leets.crazyform.domain.workspace.exception.WorkspaceNotFoundException;
+import leets.crazyform.domain.workspace.repository.WorkspaceRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class WorkspaceDeletionImpl implements WorkspaceDeletion {
+    private final WorkspaceRepository workspaceRepository;
+
+    @Transactional
+    @Override
+    public void deleteWorkspace(UUID workspaceId) {
+        if (!workspaceRepository.existsById(workspaceId)) {
+            throw new WorkspaceNotFoundException("Workspace not found with ID: " + workspaceId);
```

Comment:
```suggestion
            throw new WorkspaceNotFoundException();
```

위 Exception 리뷰에서 별도의 인자를 사용하지 않기 때문에, 여기서도 인자를 넘겨줄 필요가 없습니다.